docs/docs/references/configuration/cli/trivy_filesystem.md

```diff
@@ -46,6 +46,7 @@ trivy filesystem [flags] PATH
       --include-dev-deps                 include development dependencies in the report (supported: npm, yarn)
       --include-non-failures             include successes and exceptions, available with '--scanners config'
       --java-db-repository string        OCI repository to retrieve trivy-java-db from (default "ghcr.io/aquasecurity/trivy-java-db")
+      --keep-system-installed-files      keep system installed files in analysis result output
```

Comment:
do we need to further document the usage of the flag?

Reply:
I changed it to`--retain-system-installed-files     retains the files installed by each package in the analysis output when set to true.`
Do we need to add anything ?